src/tests/operate.test.js

```diff
@@ -0,0 +1,35 @@
+import { render } from '@testing-library/react';
+import operate from '../logic/operate';
+import '@testing-library/jest-dom';
+import Calculator from '../components/Calculator';
+
+beforeEach(() => {
+  render(<Calculator />);
```

Comment:
Use the React Testing Library to simulate user interaction. There are warnings in the console and as a result, the UI is not been simulated. This is the warning: `Warning: CalcButton: key is not a prop. Trying to access it will result in undefined being returned.` Please make the required changes.

Reply:
Well done